cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/instance/StopStartDownscaleGetRunningInstancesResult.java

```diff
@@ -0,0 +1,44 @@
+package com.sequenceiq.cloudbreak.cloud.event.instance;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+
+public class StopStartDownscaleGetRunningInstancesResult extends CloudPlatformResult {
+
+    private final List<CloudInstance> runningInstancesWithServicesNotRunning;
+
+    private final String hostGroupName;
+
+    private final Set<Long> hostIds;
```

Comment:
Is this primarily to propagate the initial request to the next stage, since this information would otherwise be lost ?